package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "zendesk",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "build": "tsc -p .",
+    "build:watch": "tsc -w -p .",
+    "start": "nodemon dist/src/index.js",
+    "test": "jest --coverage",
+    "rebuild": "rm -rf node_modules; rm package-lock.json; npm i; npm run build"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "body-parser": "^1.19.0",
```

Comment:
Can we use exact package versions to be consistent with other JS repositories we have? Also add an `.npmrc` file like https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/webapp/.npmrc to enforce this.

Reply:
done.